Assembly-CSharp/ModLoader.cs

```diff
@@ -57,6 +70,8 @@ public static IEnumerator LoadModsInit(GameObject coroutineHolder)
         {
             if (Loaded || Preloaded)
```

Comment:
This entire check is redundant now with it having already been guarded against in `OnScreenDebugInfo`